### DISCUSSION TITLE
How to use podman to run harbor instead of docker?

### DISCUSSION THREAD

**Discussion by thuanghai** [goharbor/harbor#16539] [Q&A]
I don't want to install docker-ce and docker-compose. On my rocky 8 host, I want to use podman to run harbor instead of docker. Please tell me how can I do that?

**Comment by BRQ-net** [comment]
Hi,
Hope this link answers your question.
[https://www.redhat.com/sysadmin/podman-docker-compose](https://www.redhat.com/sysadmin/podman-docker-compose)

**Reply by ivansmm** [reply]
I tried this and it did not work for me since podman compose (at least the version under debian/testing) does not resolve container names as hostnames for containers - on which docker configurtion relies. I have configured harbor components to run in a single pod using localhost for communication, but I had to change port numbers per component. Resulting configuration worked, it allowed pushing and pulling images, but not searching through register. But the latter is a different problem, it it lso occurs to me even when I run harbor with default configuration under docker.

**Reply by infotechswiss** [reply]
My Issue was resolved with this: https://github.com/goharbor/harbor/issues/21914
Also runs perfectly on Podman with this setup 🥇

**Comment by dioguinhocst** [comment]
@thuanghai have you succeeded ? ..

**Comment by infotechswiss** [comment]
Just in case, here is the original URL for the Podman deployment: https://code.swissmakers.ch/michael.reber/harbor-podman
